app/shared/event/event_logger.py

```diff
@@ -0,0 +1,13 @@
+from event import Event
+
+
+class EventLogger():
+    def __init__(self):
+        pass
```

Comment:
if `__init__` will be filled with code in the future, prefer ellipsis to pass.
If not, then delete the `__init__` method.

Reply:
@Vikka connait pas ellipsis, tu peux détailler ? Quelle différence ?

Reply:
Et si `__init__` n'est pas implémenter, autant ne pas le mettre non ? on l'ajoutera quand on en aura besoin.

Reply:
Ellipsis est `...`. Pas le caractère utf8, juste '.'*3.

Ellipsis est généralement reconnu comme préférable comme marqueur "WIP", tandis que pass comme "nothing to do here". Mais ellipsis étant très peu connu / utilisé, il n'y a pas de concensus solide ni de norme. Cette sémantique de "required, but not set yet" est quelque chose que j'ai compris à force de lire et consulter des échanges sur Python, Pythonic et le fonctionnement de python.

Reply:
> Et si `__init__` n'est pas implémenter, autant ne pas le mettre non ? on l'ajoutera quand on en aura besoin.

C'est ce que je dis ici : "If not, then delete the __init__ method.". Donc je plussoie.